usr/share/rear/conf/default.conf

```diff
@@ -1529,10 +1529,19 @@ BACKUP_PROG_ARCHIVE="${BACKUP_PROG_ARCHIVE:-backup}"
 # that is used e.g. in 'tar -X backup-exclude.txt' to get things excluded from the backup.
 # Quoting of the BACKUP_PROG_EXCLUDE array members avoids bash pathname expansion
 # when bash pathname expansion is not wanted for the BACKUP_PROG_EXCLUDE array members.
+# Via the '/directory/*' form the plain directory is still included in the backup
+# which is mandatory when directories have special owner,group,permissions set
+# to ensure directories get restored from the backup with right owner,group,permissions.
+# By default /var/tmp/rear.* is excluded because ReaR uses /var/tmp/rear.* as TMPDIR
+# (see TMPDIR and KEEP_BUILD_DIR above) to avoid that one gets ReaR's whole BUILD_DIR
+# at least of the current "rear mkbackup" run included in the backup.
+# Also ReaR's VAR_DIR/output is excluded because in particular with OUTPUT=ISO
+# the ReaR recovery system ISO image var/lib/rear/output/rear-HOSTNAME.iso
+# could be rather big and is not needed in the backup.
 # In /etc/rear/local.conf use BACKUP_PROG_EXCLUDE+=( '/this/*' '/that/*' )
 # to specify your particular items that should be excluded from the backup in addition to what
 # gets excluded from the backup by default here (see also BACKUP_ONLY_EXCLUDE below):
-BACKUP_PROG_EXCLUDE=( '/tmp/*' '/dev/shm/*' "$VAR_DIR/output/*" )
+BACKUP_PROG_EXCLUDE=( '/tmp/*' '/dev/shm/*' '/var/tmp/rear.*' "$VAR_DIR/output/*" )
```

Comment:
Maybe exclude `$VAR_DIR` altogether?

Reply:
I don't know the reason why only VAR_DIR/output/ is excluded.

Here I only tried to explain why VAR_DIR/output/ is excluded
as far as I could find out with
```
# git log -p --follow usr/share/rear/conf/default.conf

...

commit 26c1dc702aadde3266e7df943f40e0536300da4e
Author: Gratien D'haese <gratien.dhaese@gmail.com>
Date:   Wed Apr 23 16:46:12 2014 +0200

    added $VAR_DIR/output/\* to BACKUP_PROG_EXCLUDE to avoid ISOs to being backuped up
```
that leads to
https://github.com/rear/rear/commit/26c1dc702aadde3266e7df943f40e0536300da4e

Reply:
I think I know now why only VAR_DIR/output/ is excluded:

In my last test in
https://github.com/rear/rear/pull/3175#issuecomment-2114478980
i.e. therein my test with `BACKUP_PROG_EXCLUDE+=( '/var/tmp/rear.*' /.snapshots )`
where this pull request here was not yet merged
I got in my backup.tar.gz in particular this
```
drwxr-xr-x root/root           0 2024-05-16 10:54 root/rear.lzaoral-backup-mounted-btrfs-subvolumes/var/lib/rear/
drwxr-xr-x root/root           0 2024-05-16 10:55 root/rear.lzaoral-backup-mounted-btrfs-subvolumes/var/lib/rear/output/
drwxr-xr-x root/root           0 2024-05-16 10:54 root/rear.lzaoral-backup-mounted-btrfs-subvolumes/var/lib/rear/recovery/
-rw-r--r-- root/root        1033 2024-05-16 11:06 root/rear.lzaoral-backup-mounted-btrfs-subvolumes/var/lib/rear/recovery/directories_permissions_owner_group
-rw-r--r-- root/root         750 2024-05-16 11:06 root/rear.lzaoral-backup-mounted-btrfs-subvolumes/var/lib/rear/recovery/diskbyid_mappings
-rw-r--r-- root/root         210 2024-05-16 11:06 root/rear.lzaoral-backup-mounted-btrfs-subvolumes/var/lib/rear/recovery/mountpoint_device
-rw-r--r-- root/root           6 2024-05-16 11:06 root/rear.lzaoral-backup-mounted-btrfs-subvolumes/var/lib/rear/recovery/bootloader
-rw-r--r-- root/root           0 2024-05-16 11:06 root/rear.lzaoral-backup-mounted-btrfs-subvolumes/var/lib/rear/recovery/initrd_modules
-rw-r--r-- root/root         129 2024-05-16 11:06 root/rear.lzaoral-backup-mounted-btrfs-subvolumes/var/lib/rear/recovery/storage_drivers
-rw-r--r-- root/root           0 2024-05-16 11:06 root/rear.lzaoral-backup-mounted-btrfs-subvolumes/var/lib/rear/recovery/if_inet6
drwxr-xr-x root/root           0 2024-05-16 11:06 root/rear.lzaoral-backup-mounted-btrfs-subvolumes/var/lib/rear/layout/
drwxr-xr-x root/root           0 2024-05-16 10:54 root/rear.lzaoral-backup-mounted-btrfs-subvolumes/var/lib/rear/layout/config/
-rw-r--r-- root/root         903 2024-05-16 11:06 root/rear.lzaoral-backup-mounted-btrfs-subvolumes/var/lib/rear/layout/config/df.txt
-rw-r--r-- root/root        1323 2024-05-16 11:06 root/rear.lzaoral-backup-mounted-btrfs-subvolumes/var/lib/rear/layout/config/files.md5sum
-rw-r--r-- root/root        1352 2024-05-16 11:06 root/rear.lzaoral-backup-mounted-btrfs-subvolumes/var/lib/rear/layout/diskdeps.conf
-rw-r--r-- root/root         690 2024-05-16 11:06 root/rear.lzaoral-backup-mounted-btrfs-subvolumes/var/lib/rear/layout/disktodo.conf
drwxr-xr-x root/root           0 2024-05-16 10:54 root/rear.lzaoral-backup-mounted-btrfs-subvolumes/var/lib/rear/layout/lvm/
-rw-r--r-- root/root        6443 2024-05-16 11:06 root/rear.lzaoral-backup-mounted-btrfs-subvolumes/var/lib/rear/layout/disklayout.conf
drwxr-xr-x root/root           0 2024-05-16 10:54 root/rear.lzaoral-backup-mounted-btrfs-subvolumes/var/lib/rear/sysreqs/
-rw-r--r-- root/root         860 2024-05-16 11:06 root/rear.lzaoral-backup-mounted-btrfs-subvolumes/var/lib/rear/sysreqs/Minimal_System_Requirements.txt
```
I think it is good to have the contents of
var/lib/rear/recovery/ and var/lib/rear/layout/
in the backup to be more on the safe side by default.
For example in the running ReaR recovery system
one could get var/lib/rear/layout/disklayout.conf
from the backup in certain emergency cases.
Currently I don't know a real use case
but I think it won't hurt to have those files
also in the backup.
On my original test system of
https://github.com/rear/rear/pull/3175#issuecomment-2114478980
I have
```
# du -hsc var/lib/rear/recovery var/lib/rear/layout/
20K     var/lib/rear/recovery
24K     var/lib/rear/layout/
44K     total
```
so 44K more in the backup does not really matter.

Reply:
I also like having the layout part included in backup.